src/integrations/ad-template/ad-template.module.ts

```diff
@@ -0,0 +1,17 @@
+import './env';
+import { DynamicModule, Module } from '@nestjs/common';
+import { AdTemplateService } from './ad-template.service';
+import { EntitiesModule } from '../../entities/entities.module';
+import { ConfigModule } from '../../config/config.module';
+import { BluetoothModule } from '../bluetooth/bluetooth.module';
+
+@Module({})
+export default class AdTemplateModule {
```

Comment:
I think it would be good to change the name to something that includes what "ads" refer to - just to make it more clear that this is about Bluetooth Low Energy. Maybe it could become `BleAdTemplateModule`? Note that this change would affect all the other class names, config property name and folder name though.